internal/driver/glfw/menu.go

```diff
@@ -9,16 +9,29 @@ func buildMenuOverlay(menus *fyne.MainMenu, c fyne.Canvas) fyne.CanvasObject {
 		fyne.LogError("Main menu must have at least one child menu", nil)
 		return nil
 	}
+
+	menus = addMissingQuit(menus)
+	return NewMenuBar(menus, c)
+}
+
+func addMissingQuit(menus *fyne.MainMenu) *fyne.MainMenu {
 	var firstItem *fyne.MenuItem
```

Comment:
I know it's not strictly in your code, but I was confused by this variable - could it be renamed to `lastItem`?

Reply:
Yes, gladly. I tripped a few times over it, too 😄